tests/Api/Responses/Expected/admin/sales_statistics/get_sales_statistics_response.json

```diff
@@ -0,0 +1,57 @@
+{
+    "salesPerPeriod": [
+        {
+            "period": "2023-01-01 00:00:00",
+            "total": 0
+        },
+        {
+            "period": "2023-02-01 00:00:00",
+            "total": 0
+        },
+        {
+            "period": "2023-03-01 00:00:00",
+            "total": 0
+        },
+        {
+            "period": "2023-04-01 00:00:00",
+            "total": 0
+        },
+        {
+            "period": "2023-05-01 00:00:00",
+            "total": 0
+        },
+        {
+            "period": "2023-06-01 00:00:00",
+            "total": 0
+        },
+        {
+            "period": "2023-07-01 00:00:00",
+            "total": 0
+        },
+        {
+            "period": "2023-08-01 00:00:00",
+            "total": 0
+        },
+        {
+            "period": "2023-09-01 00:00:00",
+            "total": 0
+        },
+        {
+            "period": "2023-10-01 00:00:00",
+            "total": 0
+        },
+        {
+            "period": "2023-11-01 00:00:00",
+            "total": 0
+        },
+        {
+            "period": "2023-12-01 00:00:00",
+            "total": 19500
```

Comment:
Is this date intentional or will we get failing tests when the month changes?
Also would be nice to have at least one order in any other month.